test/Npgsql.Tests/NpgsqlParameterCollectionTests.cs

```diff
@@ -76,7 +76,7 @@ public void Hash_lookup_parameter_rename_bug()
     [Test]
     public void Remove_duplicate_parameter([Values(LookupThreshold, LookupThreshold - 2)] int count)
     {
-        if (_compatMode == CompatMode.CaseSensitive)
+        if (_compatMode == CompatMode.OnePass)
```

Comment:
With many tests existing early on one mode, maybe better to just have two test classes? Like have one-pass (default) here and two-pass in NpgsqlParameterCollectionTwoPassTests?

Reply:
Many will go again :)